internal/audit/decrypt_test.go

```
@@ -0,0 +1,415 @@
+// --- Issue #41 exit criteria tests ---
+
+func TestDecryptFileEnvelopeMode(t *testing.T) {
+	dir := t.TempDir()
+	kek := bytes.Repeat([]byte{0x2a}, 32)
+
+	path := writeEnvelopeAuditFile(t, dir, kek)
+	f, err := os.Open(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)